docker/ssh_config

```diff
@@ -0,0 +1,2 @@
+Host *
```

Comment:
Why is this necessary?

Reply:
This is to avoid the "The authenticity of host '????' can't be established" question when using ssh via python.

Reply:
Could we perhaps poke them into ~/.ssh/known_hosts then? Rather than just trusting our network?